src/lib/common/globals.h

```diff
@@ -283,7 +283,7 @@ extern int64_t parse8601(const std::string& s);
 * This is common code for Duration and Throttling (at least)
 *
 */
-int64_t parse8601Time(const std::string& s);
+extern int64_t parse8601Time(const std::string& s);
```

Comment:
Not directly related with this PR, but I have take the oportunity to add `extern` to some functions that  should have it.

NTC (just informative)